### PR TITLE
Beautify code

### PR DESCRIPTION
**Pull request by ctjhoa**
Few things you should consider:
- use `let` instead of `var`
- according to aurelia sample app filename should be dashed not camel case

### CODE REVIEW

File: src/app.fetch-httpClient.config.js [hunk 1]
```diff
@@ -1,46 +1,46 @@
-import {HttpClient} from 'aurelia-fetch-client'
-import {Authentication} from './authentication';
-import {BaseConfig} from './baseConfig'
-import {inject} from 'aurelia-framework';
-import {Storage} from './storage';
+import { HttpClient } from 'aurelia-fetch-client';
```
Comment (RWOverdijk) [inline src/app.fetch-httpClient.config.js]: I don't agree with the spaces.

**Comment by RWOverdijk** [comment]
@ctjhoa 
* **Regarding let:** I agree.
* **Regarding filenames**: I partially agree. I think that's a matter of preference.  But I'll concede.

What happens when you throw this through `gulp lint`?

File: src/app.fetch-httpClient.config.js (hunk 1 above)
Comment (ctjhoa) [inline src/app.fetch-httpClient.config.js]: fixed

**Comment by ctjhoa** [comment]
@RWOverdijk  I am not sure to understand your question.

If you ask "how I did that?"
With good editor (spacemacs) performing flycheck linting and manual js-beautify with plugin.

If you ask "how automated that?"
You have several choices:
- Could be done with pre-commit hook with js-beautify
- Or with a gulp task

**Comment by RWOverdijk** [comment]
@ctjhoa I prefer adding lint to the release step, like so: https://github.com/SpoonX/aurelia-orm/blob/master/build/tasks/prepare-release.js#L30

I'll take a look at this later today, thanks!

File: src/app.httpClient.config.js [hunk 2]
```diff
@@ -3,35 +3,35 @@ import {BaseConfig}  from './baseConfig';
...
+  }
 
-					if (this.config.authHeader && this.config.authToken) {
-						token = `${this.config.authToken} ${token}`;
-					}
+  configure() {
+    RequestBuilder.addHelper('authTokenHandling', () => {
+      return (client, processor, message) => {
+        if (this.auth.isAuthenticated() && this.config.httpInterceptor) {
+          var tokenName = this.config.tokenPrefix ? `${this.config.tokenPrefix}_${this.config.tokenName}` : this.config.tokenName;
```
Comment (RWOverdijk) [inline src/app.httpClient.config.js]: Seeing how this is now a oneliner, it's a bit long. Could you change it into:

```javascript
var tokenName = (this.config.tokenPrefix ? `${this.config.tokenPrefix}_`) + this.config.tokenName;
```

File: src/authFilter.js [hunk 3]
```diff
@@ -2,4 +2,4 @@ export class AuthFilterValueConverter {
   toView(routes, isAuthenticated) {
     return routes.filter(r => r.config.auth === undefined || r.config.auth === isAuthenticated);
   }
-}
+};
```
Comment (RWOverdijk) [inline src/authFilter.js]: There's no need for this. Anywhere. Export class doesn't require a semicolon

File: src/app.httpClient.config.js [hunk 4]
```diff
@@ -3,35 +3,35 @@ import {BaseConfig}  from './baseConfig';
...
+
+    this.http.configure(x => {
+      x.authTokenHandling();
       x.withHeader('Accept', 'application/json');
-		});	
-	}
-}
+    });
+  }
+};
```
Comment (RWOverdijk) [inline src/app.httpClient.config.js]: There's no need for the semicolon.

**Comment by ctjhoa** [comment]
@RWOverdijk done

**Comment by RWOverdijk** [comment]
@ctjhoa Thanks! :)

File: src/authService.js [hunk 5]
```diff
@@ -4,113 +4,119 @@ import {Authentication} from './authentication';
 import {BaseConfig} from './baseConfig';
 import {OAuth1} from './oAuth1';
 import {OAuth2} from './oAuth2';
-import authUtils from './authUtils';
+import {authUtils} from './authUtils';
```
Comment (RWOverdijk) [inline src/authService.js]: You messed me over by doing this.

**Pull request merged**